src/v/cloud_storage/anomalies_detector.cc

```diff
@@ -252,7 +252,10 @@ bool anomalies_detector::should_stop() const {
     }
 
     if (archival::run_quota_t{_result.ops} > _received_quota) {
```

Comment:
Is there any particular rationale why we allow this to be a softly enforced quota, when it isn't softly enforced for other jobs. Is it because it's expected to be lighter weight? Fine with it either way, just curious

Reply:
I think the adjacent segment merger is the only job that strictly enforces its quota. Mostly because it doesn't do a lot of requests anyway so it's easier to achieve. Quota is shared between jobs, so the total overshoot is given by that of the last job. For the scrubber it's a few requests max. The purger is a bit worse behaved.